application/web/index.html

```diff
@@ -63,6 +63,10 @@ <h2 id="journal-title"></h2>
 
                 </div>
 
+                <div class="no-entry-text">
+                    <p>You have no Journal Entries</p>
```

Comment:
Can you make this text a bit bolded. I think you can do font-weight: 500? Make sure it's a bit bolded than the bottom text